### ISSUE TITLE
Description ambiguity in index.js configuration section

### ISSUE THREAD

**Issue by daneekadoc** [mark-mucchetti/bq-location#2]
There should be a comment next to 

defaultLocation = "US" ;

inside index.js, because this location must indicate the geographical zone of the database, not the bucket (in my case it is in Europe). most likely it can be left on the initial value (US), because BigQuery will place its databases in this region. Changing this value will make the routine fail with a "Not found: Dataset {your dataset's name here}:{database name here}" error.

**Comment by mark-mucchetti** [comment]
Good catch. I tried to avoid US-centricity in all of the samples, so thank you for testing it and finding the problem.
